bootstrap/pkg/kfapp/coordinator/coordinator.go

```diff
@@ -457,6 +471,65 @@ func (kfapp *coordinator) Apply(resources kftypes.ResourceEnum) error {
 	return nil
 }
 
+func (kfapp *coordinator) applyCommon() error {
+
+	// Get a K8s client
+	config := kftypes.GetConfig()
+	kubeclient := kftypes.GetClientset(config)
+
+	// Create KFApp's namespace
+	ns := &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: kfapp.KfDef.Namespace,
+		},
+	}
+	log.Infof("Creating namespace: %v", ns.Name)
+
+	_, err := kubeclient.CoreV1().Namespaces().Create(ns)
+	if err != nil && !errors.IsAlreadyExists(err) {
+		log.Errorf("Error creating namespace %v", ns.Name)
+		return &kfapis.KfError{
+			Code:    int(kfapis.INTERNAL_ERROR),
+			Message: err.Error(),
+		}
+	}
+
+	// Apply Istio if use_istio option is set
+	if kfapp.KfDef.Spec.UseIstio {
```

Comment:
I think installing istio when --platform=None will cause problems.

I think lots of people will want to install KF on their existing cluster which already has ISTIO installed and don't want us to mess with it; see for example
https://github.com/kubeflow/kubeflow/issues/3396